<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vega sPro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icons/media-icon-window.png</normaloff>:/images/icons/media-icon-window.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widgetVideos" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widgetList" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext"/>
   <addaction name="actionRemove"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/folder.png</normaloff>:/images/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a File</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/media-play.png</normaloff>:/images/icons/media-play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/media-pause.png</normaloff>:/images/icons/media-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause current video</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/media-step-backward.png</normaloff>:/images/icons/media-step-backward.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous video</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/media-step-forward.png</normaloff>:/images/icons/media-step-forward.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next video</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/media-remove.png</normaloff>:/images/icons/media-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove from playlist</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
